src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/Utils.scala

```diff
@@ -188,15 +190,10 @@ object Utils {
 
     val organization: util.Map[String, AnyRef] = new util.HashMap()
     organization.put("id", tenantId.toString)
-    organization.put("name", SUPPORT_ROLE)
+    organization.put("name", PAGOPA)
     organization.put("roles", List(role).asJava)
 
-    Map(
-      USER_ROLES            -> SUPPORT_ROLE,
```

Comment:
I double checked in commons, this field is required to identify the user role inside interop and must be kept